### DISCUSSION TITLE
Sum of transactions not matching balance

### DISCUSSION THREAD

**Discussion by salvationarinze** [bavix/laravel-wallet#717] [General]
For certain operations, I credit a bunch of wallets and some wallets get multiple deposits. I just discovered that for some of my users, the sum in the tnxs doesn't the wallet balance. I wonder what might be going on

**Comment by rez1dent3** [comment]
Hello. Unable to answer the question due to lack of configuration data (php, database, cache, lock, wallet configuration, laravel version ).
